experiments.py

```diff
@@ -68,9 +68,9 @@ def save_results_dict(arguments, results_dict, f_name_prefix = ""):
             rc_dir_str = "results/{}/{}/{}/{}".format(arguments["experiment name"],
                 arguments["dataset name"], arguments["base model"]["name"],recourse_name).replace(" ", "")
             pathlib.Path(rc_dir_str).mkdir(parents=True, exist_ok=True) 
-            recourse_results[recourse_name].to_csv(rc_dir_str+'/'+f_name_prefix+'trial_results.csv', index=False)
-            agg_recourse_results[recourse_name].to_csv(rc_dir_str+'/'+f_name_prefix+'agg.csv', index=False)
-            agg_recourse_results_err[recourse_name].to_csv(rc_dir_str+'/'+f_name_prefix+'agg_err.csv', index=False)
+            recourse_results[recourse_name].to_csv(rc_dir_str+'/'+f_name_prefix+'all_trials_results.csv', index=False)
+            agg_recourse_results[recourse_name].to_csv(rc_dir_str+'/'+f_name_prefix+'agg_all_trials_results.csv', index=False)
```

Comment:
Strongly suggest combining these into a single file to reduce file overload and to contextualize the aggregated results.

Add a column for the recourse name (i.e. "StEP", "DiCE", "FACE", "C-CHVAE") in the aggregated recourse results file.

For each metric, you could do <mean +/- stderr>.